app/controllers/cards.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from 'express';
+import { AxiosResponse } from 'axios';
+import HttpStatus from 'http-status-codes';
+import { getAllCards as getAllCardsService, getInfo as getInfoService } from '../services/cards';
+import { Card, Info } from '../services/cards/types';
+
+export function getAllCards(req: Request, res: Response, next: NextFunction): Promise<Response | void> {
+  return getAllCardsService()
+    .then((cards: AxiosResponse<Card[]>) => res.status(HttpStatus.OK).send(cards.data))
```

Comment:
I think that the service should return directly the cards and not the axios response